javascript-loops-practice/filter-out-nulls.js

```diff
@@ -0,0 +1,12 @@
+/* exported filterOutNulls */
+function filterOutNulls(values) {
+  const newArray = [];
+  for (let i = 0; i < values.length; i++) {
+    if (values[i] === null) {
+      delete values[i];
+    } else {
+      newArray.push(values[i]);
+    }
```

Comment:
We don't really want to `delete values[i]`, because then you're manipulating the array that gets passed in. If the value is equal to `null`, we don't want to do anything. So the best way to go about this, would be using the logical not operator to check that it's NOT null.

```
if (values[i] !== null) {
  newArray.push(values[i])
}
```